source/pay-product-page/javascripts/cookie-banner/test-banner-template.js

```diff
@@ -0,0 +1,28 @@
+module.exports = `
+    <div id="pay-cookie-banner" class="pay-cookie-banner govuk-width-container" data-module="pay-cookie-banner" role="region" aria-describedby="pay-cookie-banner__heading">
```

Comment:
Using this template for cookies banner tests ensures events (show/hide banner, set cookies,..) are working as expected. 
if content is changed in `source/layouts/layout.erb` and html tags have different classes, tests will still but cookies banner won't want. Would be nice if we can parse the actual html used it makes any changes can be detected.

May be extract cookie banner content to html and embed in layout.erb (if possible)